tests/FSharp.Data.GraphQL.Tests/IntrospectionTests.fs

```diff
@@ -21,15 +20,119 @@ type IntrospectionData = {
     Data:  IntrospectionResult
 }
 
-
-[<Fact(Skip="FIXME: investigate reason of failure")>]
+[<Fact>]
 let ``Introspection schema should be serializable back and forth using json`` () =
     let root = Define.Object("Query", [ Define.Field("onlyField", String) ])
     let schema = Schema(root)
-    let introResult = Executor(schema).AsyncExecute(Introspection.introspectionQuery) |> sync
-    let json = toJson introResult
-    let deserialized:IntrospectionData = Helpers.fromJson json
-    deserialized.Data.__schema |> equals (schema :> ISchema).Introspected
```

Comment:
This test was not working mainly because of two factors:

1. The introspection query inside `Introspection.introspectionQuery` does not ask for all fields that are serialized.
2. The `equals` comparison was being made with the `GQLResponse` itself, which is not serialized at all.